app/src/main/java/com/into/websoso/ui/login/LoginActivity.kt

```
@@ -5,153 +5,49 @@ import android.content.Intent
-                    }
-                }
-
-                is LoginUiState.Idle -> {
-                    binding.wllLogin.visibility = View.INVISIBLE
-                }
+        enableEdgeToEdge()
+        setContent {
+            WebsosoTheme {
+                SignInScreen(
+                    authClient = { platform ->
+                        authClient[platform] ?: throw IllegalStateException()
```

detekt (1.23.7)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)